src/pymatgen/analysis/pourbaix_diagram.py

```diff
@@ -62,12 +66,15 @@
 
 logger = logging.getLogger(__name__)
 
-PREFAC = 0.0591
+PREFAC: float = 0.0591
```

Comment:
I guess it would be better to use to original $\frac{RT}{nF}$ instead of rounded one?